### PR TITLE
Initial Nightwatch JS tests

### PR DESCRIPTION
**Pull request by matharden**
Initial a set-up of Nightwatch browser tests, which can later be run through [BrowserStack](https://www.browserstack.com/).
## Stories covered
- [As a developer I want to generate browser testing framework](https://www.pivotaltracker.com/story/show/94952414) - 94952414

### CODE REVIEW

File: app/views/static/index.html.haml [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 %ul.SummaryBoxes
   %li.SummaryBoxes--fresh
-    %a(href="#TODO")
+    %a.quick(href="#TODO/new")
```
Comment (SteveMarshall) [inline app/views/static/index.html.haml:5]: Ooooh, I like this. Very nice.

File: nightwatch [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+require('nightwatch/bin/runner.js');
```
Comment (SteveMarshall) [inline nightwatch]: Can this go into `bin`?

File: tests/nightwatch/home.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  'Home page provides links to view new and processed credits' : function (browser) {
+    browser
+      .url('http://localhost:3000/')
```
Comment (SteveMarshall) [inline tests/nightwatch/home.js:4]: Did you intend for the port on this URL to be different to the one in `tests/nightwatch.json`?

File: nightwatch (hunk 2 above)
Comment (matharden) [inline nightwatch]: Possibly… good shout.

File: tests/nightwatch/home.js (hunk 3 above)
Comment (matharden) [inline tests/nightwatch/home.js:4]: Hmmm… lemme check this…

Comment (matharden) [inline tests/nightwatch/home.js:4]: Removed from config JSON.

**Pull request merged**